Source/Core/Core/NetPlayClient.cpp

```diff
@@ -1534,6 +1566,30 @@ bool NetPlayClient::WiimoteUpdate(int _number, u8* data, const u8 size, u8 repor
   return true;
 }
 
+void NetPlayClient::HostPadPoll()
+{
+  if (m_local_player->pid == 1)
```

Comment:
```cpp
if (m_local_player->pid != 1)
    return;
```

If nothing else is planned to be added to this function. It would likely be more self-documenting of the significance of pid 1 if a member function like:

```cpp
bool IsSomeDescriptiveName() const
{
    return pid == 1;
}
```

was added to the `Player` class, which could be called instead of spreading `pid == 1` in (currently) two places.